api/src/main/resources/api/service_openapi.yaml

```diff
@@ -1124,24 +1130,30 @@ components:
 
     DisplayHintEnumV2:
       type: object
-      description: Enumeration of possible values and display strings
+      description: Enumeration of possible values, display strings, and counts
       properties:
         enumHintValues:
           type: array
           items:
-            $ref: "#/components/schemas/ValueDisplayV2"
+            type: object
+            description: Enum value and count
+            properties:
+              enumVal:
+                $ref: "#/components/schemas/ValueDisplayV2"
+              count:
+                type: integer
 
     DisplayHintNumericRangeV2:
       type: object
       description: Maximum and minimum values
       properties:
         min:
           description: Max value
-          type: integer
+          type: number
           format: int64
```

Comment:
What does `format: int64` mean after this change? Can it just be removed?

Reply:
Hmm, I think in this case it doesn't matter to the service because this is an output schema. But for input schemas that the service reads, the format indicates whether to pull the value as a Java long (int64)/integer (int32)/float/double.

Changed this to double, since that's what we're outputing.